### ISSUE TITLE
Mismatched type error when trying to run the example: expected `Vec<Vec3>`, found `Vec<(UVec2, u16)>`

### ISSUE THREAD

**Issue by nylira** [TheGrimsey/oxidized_navigation#8]
Hey there, I was trying to run the example code on bevy 0.10.1 and I ran into this issue with the following line when trying to run it: https://github.com/TheGrimsey/oxidized_navigation/blob/15646624ece7e8b3387f1feedc76ff0d4da3efab/examples/blocking_async.rs#L202

```rust
error[E0308]: mismatched types
   --> src/systems/movement.rs:201:25
    |
201 |             return Some(path);
    |                    ---- ^^^^ expected `Vec<Vec3>`, found `Vec<(UVec2, u16)>`
    |                    |
    |                    arguments to this enum variant are incorrect
    |
    = note: expected struct `Vec<bevy::prelude::Vec3>`
               found struct `Vec<(UVec2, u16)>`
help: the type constructed contains `Vec<(UVec2, u16)>` due to the type of the argument passed
   --> src/systems/movement.rs:201:20
    |
201 |             return Some(path);
    |                    ^^^^^----^
    |                         |
    |                         this argument influences the type of `Some`
note: tuple variant defined here
   --> /home/pz/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/option.rs:571:5
    |
571 |     Some(#[stable(feature = "rust1", since = "1.0.0")] T),
    |     ^^^^

For more information about this error, try `rustc --explain E0308`.
error: could not compile `bevy_ball_game` due to previous error
```

**Comment by TheGrimsey** [comment]
The version on github is ahead of the published ``0.4.0`` version.

You should look at either pulling the repo and calling ``cargo run --example blocking_async`` or looking at the example at the release commit.

https://github.com/TheGrimsey/oxidized_navigation/blob/6af9c2d8b6c4c4fe4b7a9164fd4e89929dd727bb/examples/blocking_async.rs

**Comment by nylira** [comment]
Thank you for the tip! I'll try cloning later today. Great work on this library.
